modules/core/src/main/java/org/apache/ignite/internal/processors/cache/transactions/IgniteTxManager.java

```diff
@@ -3016,6 +3031,13 @@ else if (tx.setRollbackOnly())
                     }
                 }
 
+                timeBag.finishGlobalStage("Initialized");
```

Comment:
Let's check here - is info level enabled.
Otherwise, we perform a lot of unnecessary work - finishGlobalStage includes acquiring write lock, creating some objects, etc.

Reply:
done